api/src/main/java/com/redhat/ipaas/api/v1/model/ListResult.java

```diff
@@ -0,0 +1,17 @@
+package com.redhat.ipaas.api.v1.model;
+
+import org.immutables.value.Value;
+
+import java.util.Collection;
+
+@Value.Immutable
+public interface ListResult<T extends WithId> {
+
+    int getTotalCount();
+
+    Collection<T> getItems();
+
+    class Builder<T extends WithId> extends ImmutableListResult.Builder<T> {
```

Comment:
I can't find the `ImmutableListeResult.Builder` in the package (and its not imported)